com.seafile.Client.yaml

```diff
@@ -0,0 +1,83 @@
+app-id: com.seafile.Client
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: seafile-applet
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --persist=.ccnet
+  - --filesystem=home/Seafile:create
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=com.canonical.indicator.application
+cleanup:
+  - '*.a'
+  - '*.la'
+  - /bin/vala*
+  - /bin/vapigen*
+  - /bin/vapicheck*
+  - /include
+  - /lib/debug
+  - /lib/vala*
+  - /lib/pkgconfig
+  - /lib/libvala*
+  - /share/aclocal
+  - /share/devhelp
+  - /share/man
+  - /share/pkgconfig
+  - /share/vala*
+
+rename-desktop-file: seafile.desktop
+rename-icon: seafile
+
+modules:
+  - name: vala-bootstrap
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/vala/0.16/vala-0.16.1.tar.xz
+        sha256: 8be28ff6347feecb391220ceb4e2209d32b697278e87ed7fc6c9fb1e9345e0d8
+
+  - name: vala
```

Comment:
are you sure you need vala?

Reply:
Yes, vala is needed for sure:
https://github.com/haiwen/seafile/blob/d01b8de6d378c69525e6ed57b84bd22579b96444/configure.ac#L28

Reply:
ok. I thought that building from tarball didn't require Vala.